packages/taquito/src/operations/operation-emitter.ts

```diff
@@ -201,6 +201,13 @@ export abstract class OperationEmitter {
               ...op,
               period: currentVotingPeriod?.voting_period.index,
             };
+          case OpKind.UPDATE_CONSENSUS_KEY:
+            return {
+              ...op,
+              ...getSource(op),
+              ...getFee(op),
+              pk: op.pk,
```

Comment:
I don't think this line (`pk: op.pk`) is needed. 
`case OpKind.UPDATE_CONSENSUS_KEY` could be moved above with 
```
case OpKind.REVEAL:
case OpKind.DELEGATION:
case OpKind.REGISTER_GLOBAL_CONSTANT:
case OpKind.TX_ROLLUP_ORIGINATION:
case OpKind.TX_ROLLUP_SUBMIT_BATCH:
```